plugins/messages/app/models/message.rb

```diff
@@ -40,6 +40,13 @@ def group_id=(group_id)
     add_recipients Group.find(group_id).users unless group_id.blank?
   end
 
+  def order_id=(order_id)
+    @order_id = order_id
+    for ordergroup in Order.find(order_id).ordergroups
+      add_recipients ordergroup.users
+    end
```

Comment:
Silly me, that wouldn't work at all. Perhaps another `has_many :through` in Order would work, but let's leave it this way :o